### ISSUE TITLE
'.' in @INC no longer present

### ISSUE THREAD

**Issue by KingCuong88** [ncbi/ngs#20]
I was trying to setup NGS on my system and I was running into the error: 

> Can't locate package.prl in @INC

I found that apparently as of v5.26, (2017-05-30) the '.' in @INC is gone by default on new compilations.

Adding '.' to PERL5LIB fixes the problem for me but I'm assuming others will run into this issue.

**Comment by lianyi** [comment]
Thanks @KingCuong88 ! 

Hit the same issue on ubuntu 17.04
```
Can't locate package.prl in @INC (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/pe
rl/5.24.1 /usr/local/share/perl/5.24.1 /usr/lib/x86_64-linux-gnu/perl5/5.24 /usr/share/perl5
/usr/lib/x86_64-linux-gnu/perl/5.24 /usr/share/perl/5.24 /usr/local/lib/site_perl /usr/lib/x8
6_64-linux-gnu/perl-base) at konfigure.perl line 35.
```
Adding . to $PERL5LIB fixed it.

**Comment by klymenko** [comment]
Fixed in 2.9.0 release.
